wgpu-rs/src/lib.rs

```diff
@@ -187,6 +213,37 @@ impl Device {
         }
     }
 
+    pub fn create_bind_group(&self, desc: &BindGroupDescriptor) -> BindGroup {
+        use BindingResource::*;
+        let bindings = desc.bindings.iter().map(|binding|
+            wgn::Binding {
+                binding: binding.binding,
+                resource: match &binding.resource {
```

Comment:
shouldn't need `&` here

Reply:
Compiler recommended it. Else binding would move out of borrowed context.